### PR TITLE
Feat: go symbols compatibility

### PR DESCRIPTION
**Pull request by shaleynikov**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -81,41 +80,60 @@ function handleError(error: AxiosError) {
...
+    sample: 'samples',
+  } as Record<string, string>
+
+  const newStringTable = profile.stringTable
+    ?.slice(0, 5)
+    .map((s) => (replacements[s] ? replacements[s] : s))
+    .concat(profile.stringTable?.slice(5))
+
+  // Inject line numbers and file names into symbols table
+  const newProfile = profile.location?.reduce(
```
Comment (eh-am) [inline src/index.ts:95]: honestly i am going to need some (in code) comments here, there's too much stuff going on

Comment (shaleynikov) [reply src/index.ts:95]: Added as much comments as I can

File: src/index.ts [hunk 2]
```diff
@@ -156,10 +183,9 @@ let chunk = 0
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const writeProfileAsync = (profile: perftools.perftools.profiles.IProfile) => {
   pprof.encode(profile).then((buf) => {
-    fs.writeFile(`${config.name}-${chunk}.pb.gz`, buf, (err) => {
+    fs.writeFile(`${config.name}-${++chunk}.pb.gz`, buf, (err) => {
```
[the remark this answers is not in the document]
Comment (shaleynikov) [reply src/index.ts:186]: no it's never used getting called anywhere, left there for instrumentation reasons.

[the remark this answers is not in the document]
Comment (eh-am) [reply src/index.ts:186]: "instrumentation reasons"?

[the remark this answers is not in the document]
Comment (shaleynikov) [reply src/index.ts:186]: Yes, for debugging.

**Comment by petethepig** [comment]
Closing this since #14 has all the same commits

**Pull request merged**